elastic/kibana
# [Infra UI] Enable infra nav in serverless

## Summary

This PR adds the `Infrastructure` item to the serverless side-nav 

_infra enabled_
<img width="1200" alt="image" src="https://github.com/elastic/kibana/assets/2767137/baaf95ae-2960-4cdc-8d90-c95018d2fa24">


_infra disabled_
<img width="1200" alt="image" src="https://github.com/elastic/kibana/assets/2767137/386635b3-d095-4f62-9831-a37a723b69c6">

Stateful Kibana isn't affected by this change.

### How to test

- Start a local es instance: `yarn es serverless --kill --clean --license trial --ssl`
- Enable `infra` in the `serverless.oblt.dev.yml` file:
  - `xpack.infra.enabled: true`
- Start a local kibana instance: `yarn serverless-oblt --ssl` and see if the side nav contains the Infrastructure item
- Disable infra and check if the side nav does not contain the infrastructure item.
